chore: align README for non-prefixed packages (#2994)
vaadin/web-components: packages/accordion/README.md

# @vaadin/accordion

> A web component implementing accordion widget: a vertically stacked set of expandable panels.

Reviewer: ```suggestion
A web component for displaying a vertically stacked set of expandable panels.
```

Widget is not a good term here, also it's obvious that it's about an accordion.

Author: Thanks, changed.

> [Live Demo ↗](https://vaadin.com/docs/latest/ds/components/accordion)

Reviewer: ```suggestion
[Documentation + Live Demo ↗](https://vaadin.com/docs/latest/ds/components/accordion)
```
We should still mention the documentation, for users it's not obvious that the live demo will actually point to the docs.

Author: Updated. Will do the same for all the `vaadin-` packages in a separate PR.

</vaadin-accordion>
```

[<img src="https://raw.githubusercontent.com/vaadin/web-components/master/packages/accordion/screenshot.png" alt="Screenshot of vaadin-accordion" width="900">](https://vaadin.com/docs/latest/ds/components/accordion)

## Installation

Install the component:

```sh

> npm i @vaadin/accordion --save

Reviewer: `--save` should not be necessary 🤔

Author: Updated. Will do the same for all the `vaadin-` packages in a separate PR.

Apache License 2.0

> Vaadin collects development time usage statistics to improve this product.

Reviewer: ```suggestion
Vaadin collects usage statistics at development time to improve this product.
```

Author: Updated. Will do the same for all the `vaadin-` packages in a separate PR.
